### DISCUSSION TITLE
blocks/feature: can I show a custom link title instead of "Read more ..."

### DISCUSSION THREAD

**Discussion by rkranendonk** [google/docsy#1694] [Q&A] [answered]
The title says it all really. How can I display a different link title than the default "Read more ..."?

**Comment by fekete-robert** [comment]
You can change it globally by changing the value of the `ui_read_more` parameter in the language file. For a single block, use the `url_text` parameter, like:

```
{{% blocks/feature url_text="Hello"
```

**Reply by rkranendonk** [reply]
Thanks for helping, these are my first days with Hugo and Docsy. 

I don't seem to have a file in my project with the string `ui_read_more` in it. So I started digging in the Docsy documentation, and found this under [Create custom UI strings](https://www.docsy.dev/docs/language/#create-custom-ui-strings):

> you can create your own project-specific internationalization file in your project’s /i18n directory

I also don't have a /i18n directory.

What should be my next steps?

**Reply by fekete-robert** [reply, marked as answer]
If you have lots of feature blocks and you want to change the text for all of them globally, you can create the i18n/en.toml file, copy its contents from the main docsy repo, and change the Read more text to something you like.

But probably it's easier to add the `url_text="show-this-instead-of-read-more"` parameter where you invoke the blocks/feature shortcode.

**Reply by rkranendonk** [reply]
Thanks! Seems an [undocumented](https://www.docsy.dev/docs/adding-content/shortcodes/#blocksfeature) feature, btw.

**Comment by fekete-robert** [comment]
Yes, I think I have a PR for it somewhere, just hasn't been merged yet.
